src/index.js

```diff
@@ -14,6 +14,23 @@ const getDefaultPayload = () => ({
   },
 });
 
+const getSessionHistory = () => {
+  const history = localStorage.getItem('history');
```

Comment:
set the key to `TwylaWidget__chat_history`. Local storage could be used by the host page and we need to make sure we don't collide with their keys.